code/src/App.js

```diff
@@ -1,9 +1,42 @@
-import React from 'react'
-
-export const App = () => {
-  return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+import React from "react";
+import { Provider } from "react-redux";
+import { configureStore, combineReducers } from "@reduxjs/toolkit";
+import todoData from "reducers/todoData";
+import TodoList from "components/TodoList";
+import AddTodo from "components/AddTodo";
+import TodoCount from "components/TodoCount";
+import { CompleteBtn } from "components/CompleteBtn";
+
+
+const reducer = combineReducers({
+  todo: todoData
+});
+
+const store = configureStore({ reducer });
+
+export const App = () => (
+
+ 
```

Comment:
Remove all these spaces between App and Providers. Or is it GitHub that's messing it up?